core/cloudflow-blueprint/src/main/scala/cloudflow/blueprint/Topic.scala

```diff
@@ -28,16 +28,14 @@ object Topic {
  */
 // TODO check that topic-name is valid topic-name.
 final case class Topic(
-    name: String,
+    id: String,
     producers: Vector[String] = Vector.empty[String],
     consumers: Vector[String] = Vector.empty[String],
     kafkaConfig: Config = ConfigFactory.empty(),
-    // override for bootstrapServers
-    bootstrapServers: Option[String] = None,
-    create: Boolean = true,
     problems: Vector[BlueprintProblem] = Vector.empty[BlueprintProblem],
     verified: Option[VerifiedTopic] = None
 ) {
+  def name = Try(kafkaConfig.getString("topic.name")).getOrElse(id)
```

Comment:
(reminder for us: We might want to create a small lib with these config ops. They are repeated everywhere and in different ways.(e.g. the other idiom we have around is: `if (config.hasPath(lalala)) Some(config.getXYZ(lalala)) else None`)

Reply:
Definitely, good idea.